aries/aries_vcx_core/src/wallet/entry_tag.rs

```diff
@@ -19,15 +21,46 @@ impl Serialize for EntryTags {
         let mut map = serializer.serialize_map(Some(self.inner.len()))?;
         for tag in self.inner.iter() {
             match tag {
-                EntryTag::Encrypted(key, val) | EntryTag::Plaintext(key, val) => {
-                    map.serialize_entry(&key, &val)?
-                }
+                EntryTag::Encrypted(key, val) => map.serialize_entry(&key, &val)?,
+                EntryTag::Plaintext(key, val) => map.serialize_entry(&format!("~{}", key), &val)?,
```

Comment:
[As you said](https://github.com/hyperledger/aries-vcx/pull/1084#discussion_r1444318095) (and I agree), determining whether the tag is encrypted or not should be responsibility of the wallet implementation.

Reply:
Do I understand correctly that you expect having only a single variant `EntryTag::Tag(key, val)`?

Reply:
At this point I do think it probably would be better.

Reply:
Changed to a tuple.